Sming/Wiring/Print.h

```diff
@@ -26,12 +26,14 @@
 
 #define INITIAL_PRINTF_BUFFSIZE 128
 
+#undef printf_P
```

Comment:
You should remove this here. If you do not want name collision you have 2 options: namespaces or explicit calling e.g. this.printf_P